modules/rest-api/pages/rest-manage-log-collection.adoc

```diff
@@ -0,0 +1,249 @@
+The `GET /pools/default/tasks` method and endpoint, which are fully described in xref:rest-api:rest-get-cluster-tasks.adoc[Getting Cluster Tasks], can be used to return status on ongoing tasks, including the collection of log files.
+
+Both `POST /controller/startLogsCollection` and `POST /controller/cancelLogsCollection` require either the *Full Admin* or the *Cluster Admin* role.
+
+[#log-file-locations]
+=== Log-File Locations
+
+By default, on Linux systems, log files are saved to `/opt/couchbase/var/lib/couchbase/logs`; on MacOS, to `/Users/username/Library/Application Support/Couchbase/var/lib/couchbase/logs`; and on Windows, to `C:\Program Files\Couchbase\Server\var\lib\couchbase\logs`.
+
+== Curl Syntax
+
+[source,bourne]
+----
+curl -X POST http://<ip-address-or-domain-name>:8091/controller/startLogsCollection
+-u <username>:<password>
+-d nodes=[ '*' | <node-list> ]
+-d logRedactionLevel=[ 'partial' | 'none' ]
+-d logDir=<pathname>
+-d tmpDir=<pathname>
+-d logRedactionSalt=<string>
+-d upload-host=<upload-host-name>
+-d customer=<customer-name>
+-d ticket=<ticket-number>
+
+curl -X POST http://<ip-address-or-domain-name>:8091/controller/cancelLogsCollection
+-u <username>:<password>
+----
+
+For the curl syntax for `GET /pools/default/tasks`, see xref:rest-api:rest-get-cluster-tasks.adoc[Getting Cluster Tasks].
+
+Parameters for `POST /controller/startLogsCollection` are as follows:
+
+* `nodes`. Either a (quoted) wildcard character, `'*'`, indicating that logs are to be collected for all nodes in the cluster; or a list of nodes for which logs are to be collected.
+If a list is specified, each element in the list must be comma-separated from the element that follows it; and each element must take the following form:
++
+[source,bourne]
+----
+ns_1@<ip-address-or-domain-name>
+----
++
+After log-collection has started, if `POST /controller/cancelLogsCollection` is executed, log-collection is stopped on all nodes that were specified by means of the `nodes` flag.
+
+* `logRedactionLevel`.
+The redaction-level for the logs.
+This must be either `none` (the default) or `partial`.
+If `partial` is specified, data such as usernames, keys, and values are redacted from the logs.
+
+* `logDir`.
+The pathname of the directory on each node into which logs are written.
+Each node must have this pathname.
+If a node does not have the pathname, or if the directory has insufficient space, or if the directory is not accessible by user `couchbase`, log-collection fails for that node.
+
+* `tmpDir`.
+The pathname of the directory on each node into which temporary files are written, during log-collection.
+If a node does not have the pathname, the operating system's temporary directory is used instead.
+If the administrator-specified directory has insufficient space, or if the directory is not accessible by user `couchbase`, log-collection fails for that node.
+
+* `logRedactionSalt`.
```

Comment:
I don't think this should be documented. Let the value get picked automatically.

Reply:
My comment above.